spec/openapi.yaml

```diff
@@ -0,0 +1,108 @@
+openapi: 3.0.0
+info:
+  version: 1.0.0
+  title: Athenian Dashboard API
+  license:
+    name: CC-BY-4.0
```

Comment:
I think that `CC` is not a proper software license. On the other side, I know that @se7entyse7en already left this as something to be defined later, so he preferred to leave it empty (which use to mean: 'all rights reserved')

Reply:
This is the license of the API document (that's stated in the spec!), and it was the most popular according to https://matija.suklje.name/popular-licenses-in-openapi
So no worries.

Reply:
@eiso says: no licenses. So changed to proprietary (default).

Reply:
Is this a real issue? I mean the license of the spec? I just don't know whether there having a CC-BY license for the spec is in conflict with being part of an unlicensed project. \cc @eiso